internal/cmd/operator-sdk/bundle/validate/cmd.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validate
+
+import (
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle/validate/internal"
+	"github.com/operator-framework/operator-sdk/internal/flags"
+)
+
+const (
+	longHelp = `The 'operator-sdk bundle validate' command can validate both content and format of an operator bundle
+image or an operator bundle directory on-disk containing operator metadata and manifests. This command will exit
+with an exit code of 1 if any validation errors arise, and 0 if only warnings arise or all validators pass.
+
+More information about operator bundles and metadata:
+https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
+
+NOTE: if validating an image, the image must exist in a remote registry, not just locally.
+`
+
+	examples = `The following command flow will generate memcached-operator bundle manifests and metadata,
+then validate them for 'memcached-operator' version v0.1.0:
+
+  # Generate manifests and metadata locally.
+  $ make bundle
+
+  # Validate the directory containing manifests and metadata.
+  $ operator-sdk bundle validate ./bundle
+
+To build and validate an image built with the above manifests and metadata:
+
+  # Create an image registry namespace or use an existing one.
+  $ export REG_NAMESPACE=<your registry namespace>
```

Comment:
IHMO we ought add here the same quickstart steps. 
If in each place we create a new nomenclature it might start to be confusing. 
Also, in POV `NAMESPACE` for your registry can lead to misunderstanding.